Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1599,9 +1599,9 @@ CSSTextJustifyEnabled:
     WebKitLegacy:
       default: false
     WebKit:
-      default: false
+      default: true
     WebCore:
-      default: false
+      default: true
```

Comment:
inter-character/inter-word need to be implemented to enable this

Reply:
That's fine, but I wouldn't enable the preference in that case

Reply:
The status of `CSSTextJustifyEnabled` is `testable`, which means tests should have it enabled by default